chainer/link.py

```diff
@@ -551,15 +551,6 @@ def cleargrads(self):
             param.cleargrad()
 
     def zerograds(self):
-        """Initializes all gradient arrays by zero.
```

Comment:
I think it's better not to remove the existing docs for deprecated methods.
It will show up as an undocumented method in the docs, which may confuse users.